### PR TITLE
Additional component for validating continuous trajectories 

### PR DESCRIPTION
**Pull request by AtiyahElsheikh**
Dear Michael, 

Thank you for your awesome library. I was looking for something like that. 

For my testing purposes I needed an additional component and thought to share it back. 

Greetings
Atiyah

### CODE REVIEW

**Comment by AtiyahElsheikh** [comment]
Additional component for validating continuous trajectories with own unit tests included

**Comment by xogeny** [comment]
Note my comment about the typo.  Can you fix that?

**Comment by xogeny** [comment]
Oops...I failed to submit my review.  I submitted it now so you should have gotten my feedback about the typo.  (I had not used GitHub code reviews before)

File: XogenyTest/Tests/Continuous/CheckFilureEulerAgainstAnalytical.mo [hunk 1]
```diff
@@ -0,0 +1,8 @@
+within XogenyTest.Tests.Continuous;
+
+model CheckFilureEulerAgainstAnalytical "Simulate this model using Euler method"
```
[the remark this answers is not in the document]
Comment (AtiyahElsheikh) [reply XogenyTest/Tests/Continuous/CheckFilureEulerAgainstAnalytical.mo]: Yes

**Comment by AtiyahElsheikh** [comment]
Done!

https://github.com/xogeny/XogenyTest/pull/11/commits/c03b6de9426ccaa29e496d95dc7dba45b2b5d52e

**Comment by AtiyahElsheikh** [comment]
Just a small issue .. I changed the name of model via the online editor but the name of the file remained :)
 
I have already done the testing with OpenModelica (There are even OpenModelica specific annotation in the testing). I have also used the new component with the my new open-source GenKinetics library and so far it is working.

**Pull request merged**